commitizen/commands/check.py

```diff
@@ -45,30 +45,43 @@ def __call__(self):
         Raises:
             InvalidCommitMessageError: if the commit provided not follows the conventional pattern
         """
-        commit_msgs = self._get_commit_messages()
-        if not commit_msgs:
+        commits = self._get_commits()
+        if not commits:
             raise NoCommitsFoundError(f"No commit found with range: '{self.rev_range}'")
 
         pattern = self.cz.schema_pattern()
-        for commit_msg in commit_msgs:
-            if not Check.validate_commit_message(commit_msg, pattern):
-                raise InvalidCommitMessageError(
-                    "commit validation: failed!\n"
-                    "please enter a commit message in the commitizen format.\n"
-                    f"commit: {commit_msg}\n"
-                    f"pattern: {pattern}"
-                )
+        ill_formated_commits = [
+            commit
+            for commit in commits
+            if not Check.validate_commit_message(commit["msg"], pattern)
+        ]
+        displayed_msgs_content = "".join(
```

Comment:
Instead of putting "\n" at the end of the string inside, I'd try `displayed_msgs_content = ""\n.join(`. Also, it might be better if we quote `rev` and `msg` in displaying messgage.

Reply:
Thank you for your advice, I've corrected it. Please have a look.